src/math/nq.rs

```diff
@@ -31,16 +31,17 @@
  */
 
 use std::num::Float;
+use super::utils::clamp;
 
 const CHANNELS: usize = 4;
 
 const RADIUS_DEC: i32 = 30; // factor of 1/30 each cycle
 
 const ALPHA_BIASSHIFT: i32 = 10;            // alpha starts at 1
-const INIT_ALPHA: i32 = 1<<ALPHA_BIASSHIFT; // biased by 10 bits
+const INIT_ALPHA: i32 = 1 << ALPHA_BIASSHIFT; // biased by 10 bits
 
 const GAMMA: f64 = 1024.0;
-const BETA: f64 = 1.0/1024.0;
+const BETA: f64 = 1024.0.recip();
```

Comment:
I would leave it like it was, it is more obvious what it does. I also have the feeling that this function will either go away or change its name to reciproke.